src/ir/literal-utils.h

```diff
@@ -29,30 +29,8 @@ inline Expression* makeFromInt32(int32_t x, Type type, Module& wasm) {
   return ret;
 }
 
-inline bool canMakeZero(Type type) {
```

Comment:
I think it's best to keep this function as it shows a clear intent. We have pairs of "can make zero" and "make zero" in the code which look nice.

Also, we may add nondefaultable types to wasm, and this would help refactor for that.